common-lib/src/main/scala/com/gu/mediaservice/lib/ImageIngestOperations.scala

```diff
@@ -13,8 +13,18 @@ class ImageIngestOperations(imageBucket: String, thumbnailBucket: String, creden
   def storeThumbnail(id: String, file: File, mimeType: Option[String]) =
     storeImage(thumbnailBucket, fileKeyFromId(id), file, mimeType)
 
+  def storeOptimisedPng(id: String, file: File) = {
+    val key = fileKeyFromId(id)
+    storeImage(pngBucket, optimisedPngKeyFromId(id), file, Some("image/png"))
+  }
+
   def deleteOriginal(id: String) = deleteImage(imageBucket, fileKeyFromId(id))
   def deleteThumbnail(id: String) = deleteImage(thumbnailBucket, fileKeyFromId(id))
+  def deletePng(id: String) = deleteImage(pngBucket, fileKeyFromId(id))
```

Comment:
should `fileKeyFromId(id)` be `optimisedPngKeyFromId(id)` as that's how its stored above?